package.json

```
@@ -36,13 +36,15 @@
     "@edx/brand": "npm:@openedx/brand-openedx@^1.2.2",
     "@edx/browserslist-config": "1.4.0",
     "@openedx/frontend-build": "14.2.2",
-    "@openedx/paragon": "22.13.0",
+    "@openedx/paragon": "23.1.1",
```

[question]: Does it make sense to provision the latest minor version instead of a hard-coded version?

```suggestion
    "@openedx/paragon": "^23.1.1",
```

---

I made the change :)